content/applications/hr/employees/departments.rst

```diff
@@ -0,0 +1,152 @@
+  waiting for approval.
+- :guilabel:`Absence`: the number of absences for the current day.
+- :guilabel:`Color bar`: the selected color for the department appears as a vertical bar on the
+  left side of the department card.
+
+.. important::
+   When clicking on an alert in a department card, such as :guilabel:`Time Off Requests`, any other
+   alerts pending for the user to approve, including alerts from other departments, appear in the
+   list.
+
+Dashboard views
+---------------
+
+The default view for the :guilabel:`Departments` dashboard is a Kanban view. It is possible to view
+the departments in two other forms: a list view and a hierarchy view.
+
+List view
+~~~~~~~~~
+
+To view the departments in a list view, click the :icon:`fa-align-justify` :guilabel:`list` icon in
+the top-right corner. The departments appear in a list view, which displays the
+:guilabel:`Department Name`, :guilabel:`Company`, :guilabel:`Manager`, :guilabel:`Employees`,
+:guilabel:`Parent Department`, and :guilabel:`Color` for each department.
+
+The departments are sorted alphabetically by :guilabel:`Department Name`, by default.
+
+.. image:: departments/list.png
+   :align: center
+   :alt: The departments presented in a list view.
+
+Hierarchy view
+~~~~~~~~~~~~~~
+
+To view the departments in a hierarchy view, click the :icon:`fa-share-alt fa-rotate-90`
+:guilabel:`hierarchy` icon in the top-right corner. The departments appear in an organizational
+chart format, with the highest-level department at the top (typically :guilabel:`Management`), and
+all other departments beneath it. All child departments of the first-level child departments are
+folded.
+
+Each department card displays the :guilabel:`Department Name`, the :guilabel:`Manager` (and their
+profile icon), the :guilabel:`Number of Employees` in the department, and the ability to expand the
+department (:guilabel:`Unfold`) if there are child departments beneath it.
+
+Click the :guilabel:`Unfold` button on a department card to expand it. Once expanded, the
+:guilabel:`Unfold` button changes to a :guilabel:`Fold` button. To collapse the department, click
+the :guilabel:`Fold` button. Only **one** department *per row* can be unfolded at a time.
+
+.. image:: departments/hierarchy.png
+ :align: center
+ :alt: The departments presented in a hierarchy view.
+
+Click anywhere on a department card to open the department form. Click the :guilabel:`(#) Employees`
+smart button to view a list of all the employees in that department, including all employees in the
+child departments beneath it, organized by individual department.
+
+.. example::
+   In the hierarchy view, if the :guilabel:`(#) Employees` button on the :guilabel:`Management` card
+   is clicked (the highest-level department card), **all** employees appear in a list view,
+   organized by department. This is because **all** departments are children of the
```

Comment:
Let's use "grouped" instead of "organized" to be consistent with the grouping and filtering terminology across Odoo
```suggestion
   grouped by department. This is because **all** departments are children of the
```